mantle/platform/qemu.go

```
@@ -2002,8 +2025,19 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 		return nil, err
 	}
 
+	// Build the list of virtiofs mounts: regular host mounts plus systemd credential dir
+	allVirtioFSMounts := make([]HostMount, len(builder.hostMounts))
+	copy(allVirtioFSMounts, builder.hostMounts)
+	if builder.systemdCredentialDir != "" {
+		allVirtioFSMounts = append(allVirtioFSMounts, HostMount{
```

golangci-lint (2.12.2)

[error] append to slice `allVirtioFSMounts` with non-zero initialized length

(makezero)